### PR TITLE
[sailjail] Add translation catalog and keys. JB#52178

### PR DESCRIPTION
**Pull request by rainemak**

### CODE REVIEW

File: permissions/Videos.permission [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # -*- mode: sh -*-
 
-# x-sailjail-translation-catalog =
-# x-sailjail-translation-key =
+# x-sailjail-translation-catalog = sailjail-permissions
+# x-sailjail-translation-key = permission-la-videos
 # x-sailjail-description = Access Video data
```
Comment (Tomin1) [inline permissions/Videos.permission:5]: I find these names with capital letters strange. In this instance it doesn't even mean "Video" folder but "Videos". Would it be better if this was just "Access video data" or "Access videos" even. Then again it applies to only those that are in Videos folder. If there was secondary description that could explain it too. (Also how to access those on memory card..?)

Comment (rainemak) [reply permissions/Videos.permission:5]: Noticed similar errors on other descriptions. Will amend those a bit e.g. this should be just "Access videos" (without data)

File: permissions/AppLaunch.permission [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # -*- mode: sh -*-
 
-# x-sailjail-translation-catalog =
-# x-sailjail-translation-key =
+# x-sailjail-translation-catalog = sailjail-permissions
+# x-sailjail-translation-key = permission-la-app_launch
 # x-sailjail-description = Start user services
```
Comment (rainemak) [inline permissions/AppLaunch.permission:5]: This is tricky. Hopefully not broadly needed.

File: permissions/ApplicationInstallation.permission [hunk 3]
```diff
@@ -1,8 +1,8 @@
 # -*- mode: sh -*-
 
-# x-sailjail-translation-catalog =
-# x-sailjail-translation-key =
-# x-sailjail-description = Install software packages
+# x-sailjail-translation-catalog = sailjail-permissions
+# x-sailjail-translation-key = permission-la-application_installation
+# x-sailjail-description = Install applications
```
Comment (rainemak) [inline permissions/ApplicationInstallation.permission:5]: Also this is not really applications but all kind of packages. From user point of view maybe better still talk about apps.

File: permissions/ApplicationInstallation.permission [hunk 4]
```diff
@@ -1,8 +1,8 @@
 # -*- mode: sh -*-
 
-# x-sailjail-translation-catalog =
-# x-sailjail-translation-key =
-# x-sailjail-description = Install software packages
+# x-sailjail-translation-catalog = sailjail-permissions
+# x-sailjail-translation-key = permission-la-application_installation
+# x-sailjail-description = Install apps
```
Comment (rainemak) [inline permissions/ApplicationInstallation.permission:5]: This is tricky as this not really not only apps but other kind of packages as well. Maybe this better to talk about apps.

File: permissions/Location.permission [hunk 5]
```diff
@@ -1,8 +1,8 @@
 # -*- mode: sh -*-
 
-# x-sailjail-translation-catalog =
-# x-sailjail-translation-key =
-# x-sailjail-description = Access location data
+# x-sailjail-translation-catalog = sailjail-permissions
+# x-sailjail-translation-key = permission-la-positioning
+# x-sailjail-description = Access positioning
```
Comment (rainemak) [inline permissions/Location.permission:5]: See also https://github.com/sailfishos/sailfish-components-webview/pull/82

File: permissions/MediaIndexing.permission [hunk 6]
```diff
@@ -1,8 +1,8 @@
 # -*- mode: sh -*-
 
-# x-sailjail-translation-catalog =
-# x-sailjail-translation-key =
-# x-sailjail-description = Use Tracker
+# x-sailjail-translation-catalog = sailjail-permissions
+# x-sailjail-translation-key = permission-la-media_indexing
+# x-sailjail-description = Access media indexing
```
Comment (rainemak) [inline permissions/MediaIndexing.permission]: Should this be "Access media index"

**Review by rainemak — COMMENTED** [review]
Commeting myself as well.

**Pull request merged**